src/style.css

```diff
@@ -0,0 +1,95 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-size: 11px;
+  line-height: 15px;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+  display: flex;
+}
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice